### PR TITLE
Throw an error when rzk.yaml `include` is empty

### PR DESCRIPTION
**Pull request by aabounegm**

### CODE REVIEW

File: rzk/src/Rzk/Main.hs [hunk 1]
```diff
@@ -69,6 +70,7 @@ parseRzkFilesOrStdin = \case
       then do
         putStrLn ("Using Rzk project stucture specified in " <> rzkYamlPath)
         paths <- extractFilesFromRzkYaml rzkYamlPath
+        when (null paths) (error $ "No files found in " <> rzkYamlPath)
```
Comment (fizruk) [inline rzk/src/Rzk/Main.hs:73]: This is a weird error message to show when `include` is empty.
This seems to be the error message about missing `rzk.yaml`?

Comment (aabounegm) [reply rzk/src/Rzk/Main.hs:73]: I simply copied the same error from [`expandRzkPathsOrYaml`](https://github.com/rzk-lang/rzk/blob/dfa2a70454df25e9bf039d82eeb2d6e602ecf92b/rzk/src/Rzk/Main.hs#L57). I didn't interpret it as meaning `rzk.yaml` itself is missing, but rather that there are no files listed inside it.

Comment (fizruk) [reply rzk/src/Rzk/Main.hs:73]: I see, still I think we should do better.

Comment (aabounegm) [reply rzk/src/Rzk/Main.hs:73]: Ok, what error message would you prefer?

**Review by fizruk — APPROVED** [review]

**Pull request merged**